pantheon-bundle/src/main/java/com/redhat/pantheon/servlet/module/IncludedInAssembliesServlet.java

```diff
@@ -0,0 +1,95 @@
+package com.redhat.pantheon.servlet.module;
+
+import com.redhat.pantheon.jcr.JcrQueryHelper;
+import com.redhat.pantheon.model.module.Module;
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.api.SlingHttpServletResponse;
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.api.servlets.SlingSafeMethodsServlet;
+import org.apache.sling.servlets.annotations.SlingServletPaths;
+import org.jetbrains.annotations.NotNull;
+import org.osgi.framework.Constants;
+import org.osgi.service.component.annotations.Component;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.jcr.RepositoryException;
+import javax.servlet.Servlet;
+import javax.servlet.ServletException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Stream;
+
+import static com.redhat.pantheon.conf.GlobalConfig.DEFAULT_MODULE_LOCALE;
+import static com.redhat.pantheon.servlet.ServletUtils.paramValue;
+import static com.redhat.pantheon.servlet.ServletUtils.paramValueAsLocale;
+import static com.redhat.pantheon.servlet.ServletUtils.writeAsJson;
+import static javax.servlet.http.HttpServletResponse.SC_NOT_FOUND;
+
+/**
+ * Get operation to render a Released Module's related assembly list in JSON format.
+ * Only two parameters are expected in the Get request:
+ * 1. locale - Optional; indicates the locale that the module content is in, defaulted to en-US
+ * 2. module_id - indicates the uuid string which uniquely identifies a module
+ *
+ * The url to GET a request from the server is /api/module/assemblies
+ * Example: <server_url>/api/module/assemblies?locale=en-us&module_id=xyz
+ * The said url is accessible outside of the system without any authentication.
+ *
+ * @author Ben Radey
+ */
+@Component(
+        service = Servlet.class,
+        property = {
+                Constants.SERVICE_DESCRIPTION + "=Servlet to facilitate GET operation which accepts locale and module uuid to output module assemblies",
+                Constants.SERVICE_VENDOR + "=Red Hat Content Tooling team"
+        })
+@SlingServletPaths(value = "/api/module/assemblies")
```

Comment:
Great question! I don't think so - because this api is "what assemblies use this module" - so it is still module-centric. We will have an "/api/assembly" api in the future, which will likely include "what modules are used by this assembly" - and that will be assembly-centric, justifying the url structure.